src/modules/Dropdown/Dropdown.js

```diff
@@ -95,7 +95,7 @@ export default class Dropdown extends Component {
     /** Called with the React Synthetic Event on Dropdown blur. */
     onBlur: PropTypes.func,
 
-    /** Called with the current value on change. */
+    /** Called with the React Synthetic Event and current value on change. */
```

Comment:
**NOTE** this PR moves the dropdown to our "event first" callback argument pattern.  
See #218.